### ISSUE TITLE
[Bug 1.16.5] Crash after join in World

### ISSUE THREAD

**Issue by Rumaruka** [sp614x/optifine#6405]
## Description of Issue

Crash after join in World

1)Setup mod pack
2)Join in the created world
3)...
4)Crash

## OptiFine Version
HD_U_G7

## Fabric/Forge Version
Forge

## Log Files/Crash Reports
https://gist.github.com/Rumaruka/4c653eed9ff049efd4ebeda2f37b0dbd

**Comment by KaiAF** [comment]
This has been reported before, I am not sure if this is a optifine bug or what. There hasn't been any long term fixes found yet. The crash report suggests to me that it's a corrupted world or something, no clue.

`net.minecraft.world.storage.SaveFormat$LevelSave.func_237301_i_(SaveFormat.java:296) ~[?:?] {re:mixin,re:computing_frames,re:classloading}` It errors on world save.

You can try to see if this still happens without OptiFine

**Comment by Rumaruka** [comment]
Yes, I'm try load Minecraft without of , but world don't render blocks, I .. I don't know
